src/chess/callbacks.h

```diff
@@ -69,6 +69,8 @@ struct ThinkingInfo {
   int hashfull = -1;
   // Win in centipawns.
   optional<int> score;
+  // Number of successful TB probes (not the same as playouts ending in TB hit)
```

Comment:
Dot in the end of the sentence. :)

Reply:
That's not grammatically a sentence :) (but yes I will add it lol)